swagger/bifrost.swagger.json

```diff
@@ -2,10 +2,10 @@
   "swagger": "2.0",
```

Comment:
If you're updating this file, please check in the `.before.` file as well and the generated client code as well.